event_attribution_reporting.md

```diff
@@ -0,0 +1,136 @@
+
+When a trigger redirect is received, the browser will find the matching source with highest `attributionsourcepriority` value and generate a report. The other sources will not generate reports.
+
+If reports were sent for these sources, it would be possible for `reportingorigin` to use these other reports to remove the noise on the attribution triggering event. 
+
+
+### Single Reporting Window
+
+Attribution sources registered without a navigation will be reported at the end of their expirations, compared to the three reporting windows for navigation sources.
+
+This limits the amount of information that can be learned about the time a report was created in comparison to the [Click Through API](https://github.com/WICG/conversion-measurement-api#reporting-delay).
+
+## Privacy Considerations
+The privacy goals of this proposal are in line with those in the [Click Through Explainer](https://github.com/WICG/conversion-measurement-api#privacy-considerations).
+
+ >The main privacy goal of the API is to make linking identity between two different top-level sites difficult. This happens when either a request or a JavaScript environment has two user IDs from two different sites simultaneously.
+
+Further, the addition of event noising protects against any cross site data leakage based on `p`.
+
+
+### Less trigger side data
+
+Registering event attribution sources is not gated on a user interaction or top level navigation, allowing them to be registered more frequently and with greater ease. For example, by restricting to 1 bit of data and 1 report per source, a `reportingorigin` would need to register many more sources in order to link cross-site identity relative to the Click Through API.
+
+This is further restricted by rate limiting the usage of the API between two sites, using [reporting cooldowns](https://github.com/WICG/conversion-measurement-api#reporting-cooldown). Due to the different characteristics between classes of sources, these cooldowns should have independent limits on the number of reports of each type.
+
+The number of reporting windows is another vector which can contain trigger side information. By restricting to a [single window](#single-reporting-window), a `reportingorigin` does not receive any additional information on when in the attribution window a source was triggered.
+
+
+## Browsing history reconstruction
+
+Reporting attribution without a pre-existing navigation allows the `reportingorigin` to learn whether a given user on the source site visited the attributiondestination site at all. In the Click through API, this is not an issue because the `reportingorigin` knows a priori the user was navigating to attributiondestination.
```

Comment:
```suggestion
Reporting attribution without a pre-existing navigation allows the `reportingorigin` to learn whether a given user on the source site visited the `attributiondestination` site at all. For click-through reports, this is not an issue because the `reportingorigin` knows a priori the user was navigating to `attributiondestination`.
```